package.json

```diff
@@ -23,6 +23,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@mdi/font": "^5.9.55",
```

Comment:
if we're using the stylesheet link, is the dependency needed? Same with "material-design-icons"